src/routers/datatypeRouters/edges/complexes_proteins.ts

```diff
@@ -11,7 +11,7 @@ const proteinComplexFormat = z.object({
   source: z.string().optional(),
   source_url: z.string().optional(),
   protein: z.string().or(z.array(proteinFormat)).optional(),
-  complex: z.string().or(z.array(complexFormat)).optional()
+  complex: z.string().or(complexFormat).optional()
```

Comment:
cases like this would always return an array with a single value. I finally fixed it for many cases. There are still a few left but they are more complicated and will be easier when we start our refactoring and copying all queries from the routerEdge files into the routers.